forward_engineering/ddlProvider/ddlHelpers/columnDefinitionHelper.js

```diff
@@ -55,14 +93,30 @@ module.exports = ({ _, wrap, assignTemplates, templates, commentIfDeactivated, w
 		return type;
 	};
 
+	const processDefaultValue = (defaultValue = '') => {
+		if (!isNaN(defaultValue)) {
+			return defaultValue;
+		}
+
+		if (systemFunctions.find(it => it.startsWith(defaultValue.toUpperCase()))) {
+			return defaultValue;
+		}
+
+		if (!defaultValue.startsWith("'") || !defaultValue.endsWith("'")) {
+			return wrapInSingleQuotesIfRequired(defaultValue);
+		}
+
+		return defaultValue;
```

Comment:
It seems you can simplify it:
```suggestion
		return wrapInSingleQuotesIfRequired(defaultValue);
```